tasks/pipeline.py

```diff
@@ -831,7 +833,7 @@ def test_merge_queue(ctx):
     timestamp = int(datetime.now(timezone.utc).timestamp())
     test_main = f"mq/test_{timestamp}"
```

Comment:
nit: isn't this `test_default` because it's not necessarily "main" anymore with the `get_default_branch()`

Reply:
Renamed